mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/view.jsp

```diff
@@ -14,34 +15,100 @@
  */
 --%>
 
-<%@ include file="/init.jsp" %>
+<%@ include file="init.jsp" %>
 
 <div class="list-container">
 	<div class="header">
 		<h2>
 			<liferay-ui:message key="to-do-list" />
-
 			<i class="icon-check"></i>
 		</h2>
 	</div>
+	<div class="welcome-message">
+		<liferay-ui:message key="welcome" />
+		<%= user.getFirstName() %>
+		<img alt="<%= user.getFullName() %>" src="<%=user.getPortraitURL(themeDisplay)%>">
+	</div>
+	<liferay-ui:tabs names="tab-1,tab-2,tab-3" refresh="false" >
```

Comment:
Seriously, "tab-1,tab-2,tab-3"??

Please do not send PRs with placeholder naming.